### DISCUSSION TITLE
关于更新的问题

### DISCUSSION THREAD

**Discussion by moranfong** [tangly1024/NotionNext#2191] [Q&A]
看了官网的教程，尝试了一下还是不懂该如何更新，修改过blog.config.js

**Comment by tangly1024** [comment]
通常可以：保存一份 blog.config.js 的内容 ，然后点击更新出处的 Discard 2 commits
![image](https://github.com/tangly1024/NotionNext/assets/15920488/1ff113f4-62cd-457b-a2b5-67ea1e7d38be)

或者您提供一下 这个按钮的截图，看一下目前是什么状态

**Reply by moranfong** [reply]
![image](https://github.com/tangly1024/NotionNext/assets/111163031/193a3427-466c-4ef1-a1d6-870e118dd4d5)

**Reply by eddiehe99** [reply]
如果 Sync fork 点击后显示“This branch is not behind the upstream”就说明是最新版。

项目已配置自动更新，每天都会自动拉取更新，如果自动更新失败，应该会发一封邮件到您 GitHub 账号绑定的邮箱里。届时，根据 [GitHub 官方指南](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) 手动 sync 即可。
